home/templatetags/format_metadata_field_type.py

```diff
@@ -4,15 +4,15 @@
 
 
 @register.filter
-def format_metadata_field_type(value: dict) -> str:
+def format_metadata_field_type(value: dict) -> dict[str, str]:
     if value.get("title"):
-        title = value["title"]
+        title = value["title"].lower().replace(" ", "_")
```

Comment:
Did you check if there is another key you can use here besides `title`? I'd be surprised if there wasn't a way to directly access the raw field name from the pydantic model, as it will also be used by the functionality to generate json schemas.

Reply:
As was I. At the level that we're checking (properties), there's `title`, `description`, `examples` available. Looks to be described [here in the pydantic docs](https://docs.pydantic.dev/2.7/concepts/json_schema/#generating-json-schema)

n.b. `properties` comes from:
`entity, schema_json in entities.items` -> `for _, description in schema_json.properties.items`